common/db/db_test.go

```diff
@@ -407,18 +408,25 @@ func TestAuthConnection(t *testing.T) {
 	}
 	Convey("With an AWS or Keberos auth URI", t, func() {
 		enabled := options.EnabledOptions{URI: true}
+
 		var uri string
 		if testtype.HasTestType(testtype.AWSAuthTestType) {
-			uri = os.Getenv("MONGOD")
+			uriBytes, err := ioutil.ReadFile("../lib/MONGOD_URI")
+			if err != nil {
+				panic("Could not read MONGOD_URI file")
+			}
+			uri = string(uriBytes)
```

Comment:
I had some trouble with this part since I didn't immediately realize that env vars aren't retained across Evergreen `shell.exec` blocks.

Reading a file produced by the Evergreen script seems like the easiest way to get the connection string here.